scripts/scil_assign_color_to_trk.py

```diff
@@ -10,33 +10,37 @@
 """
 
 import argparse
+import os
 
-from dipy.io.stateful_tractogram import StatefulTractogram
-from dipy.io.streamline import save_tractogram
+from dipy.io.streamline import save_tractogram, load_tractogram
+import json
 import numpy as np
 
-from scilpy.io.streamlines import load_tractogram_with_reference
 from scilpy.io.utils import (assert_inputs_exist,
                              assert_outputs_exist,
-                             add_overwrite_arg,
-                             add_reference_arg)
+                             add_overwrite_arg)
 
 
 def _build_arg_parser():
     p = argparse.ArgumentParser(
         description=__doc__,
         formatter_class=argparse.RawTextHelpFormatter)
 
-    p.add_argument('in_tractogram',
```

Comment:
Because 1) the script name is trk and mainly 2) you can't save metadata to tck/fib/vtk (so trk only)

Reply:
I know but you can have a tck as an input.

Reply:
OK so I changed it, now it is scil_assign_color_to_tractogram.py
It can receive a tck, but always save a trk (Display a warning if this happen when using --out_suffix)